lib/web/fetch/formdata.js

```diff
@@ -157,12 +157,24 @@ class FormData {
   }
 
   [nodeUtil.inspect.custom] (depth, options) {
-    let output = 'FormData:\n'
-    this[kState].forEach(entry => {
-      output += `${entry.name}: ${entry.value}\n`
-    })
+    const state = this[kState].reduce((a, b) => {
+      if (a[b.name]) {
+        if (Array.isArray(a[b.name])) {
+          a[b.name].push(b.value)
+        } else {
+          a[b.name] = [a[b.name], b.value]
+        }
+      } else {
+        a[b.name] = b.value
+      }
+
+      return a
+    }, {})
```

Comment:
What about prototype pollution?

Reply:
doesn't matter, there are ten thousand areas vulnerable to prototype pollution

Reply:
You joking, right?

Reply:
no, we don't use primordials anywhere in the codebase (nor should we). it's a non-issue.

Reply:
oh I see what you're talking about... I thought you meant more generally